controllers/etcd_controller.go

```diff
@@ -365,6 +365,10 @@ func (r *EtcdReconciler) reconcileServices(ctx context.Context, logger logr.Logg
 		return nil, err
 	}
 
+	if etcd.Spec.Replicas > 1 {
+		svc.Spec.ClusterIP = "None"
+	}
```

Comment:
Wouldn't it work without this change? The pods would be scaled down and the service's `podSelector` will no longer match.

Reply:
I didn't understand.. Eventually, pods would be scaled up and clusterIP is for then